swagger.yaml

```diff
@@ -1,83 +1,224 @@
 swagger: '2.0'
+
 info:
-  title: Web API Skeleton
-  description: Skeleton for Dropwizard Web APIs
-  version: '0.0'
-  license:
-    name: GNU Affero General Public License Version 3
-    url: http://www.gnu.org/licenses/agpl-3.0.en.html
-externalDocs:
-  description: GitHub Repository
-  url: https://github.com/osu-mist/web-api-skeleton
-host: localhost
-schemes:
-  - http
-basePath: /api/v0
-consumes:
-  - text/plain
-produces:
-  - text/plain
+  version: "0.0.1"
+  title: Locations API
+
+# Describe your paths here
 paths:
-  /:
+  # This is a path endpoint. Change it.
+  /locations:
+    # This is a HTTP operation
     get:
-      summary: Get information
-      description: Build and runtime information about the application.
-      responses:
-        200:
-          description: Object containing information
-          schema:
-            $ref: '#/definitions/Info'
-  /sample:
-    get:
-      summary: Get message
-      description: This is a sample resource that returns a short message.
+      tags:
+       - locations
+      # Describe this verb here. Note: you can use markdown
+      description: "Gets all `Location` objects. Can be filtered by parameters."
+      operationId: getLocations
+      # This is array of GET operation parameters:
+      produces:
+        - application/json
+      parameters:
+        - in: query
+          name: q
+          description: "Query to search through buildings. Looks through the building name, abbreviation, summary and description"
+          required: false
+          type: string
+        - in: query
+          name: dining
+          description: "Specify 1 for locations that are dining areas"
+          required: false
+          type: string
+        - in: query
+          name: is_open_dining
+          description: "Specify 1 for dining locations that are currently open"
+          required: false
+          type: string
```

Comment:
Good catch. I forgot to update the parameters section. Some of these features will be implemented in the next version of the API.